lib/open_food_network/users_and_enterprises_report.rb

```diff
@@ -63,14 +65,25 @@ def users_and_enterprises
     end
 
     def sort(results)
+      pp results
```

Comment:
Yeah I added that in for debugging... still working on the last few failing specs.